src/templates/ucp.html

```diff
@@ -198,6 +198,11 @@ <h3>{{ 'Miscellaneous'|trans }}</h3>
         />
         <label for='show_public'>{{ 'Show experiments/database items from outside the team'|trans }}</label>
         <br>
+        <input id='items_steps_scope' type='checkbox' name='items_steps_scope'
```

Comment:
I don't like `items_steps_scope` as a name, because it is not obvious what it does. I suggest: `todolist_steps_show_team` instead, and why not extend this switch to include experiments and items at the same time?

Reply:
> why not extend this switch to include experiments and items at the same time

I'm not sure what you mean here.

Reply:
The user <=> team switch could work for both experiments and items. So you can see only the unfinished steps for your experiments or for other experiments in the team/with read access. So one switch will control both experiments and items scope.